src/info/author.rs

```diff
@@ -51,7 +51,7 @@ impl std::fmt::Display for Author {
         } else {
             write!(
                 f,
-                "{}% {} {}",
+                "{:3}% {} {}",
```

Comment:
```suggestion
                "{:>3}% {} {}",
```

minor, just for readability

Reply:
Also done.